ray-kubernetes/worker.yml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: ray-worker
+  labels:
+    app: ray-worker
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: ray-worker
+    spec:
+      containers:
+      - name: ray-worker
+        image: <PASTE-YOUR-IMAGE-URI>
```

Comment:
What should the image be? Should it essentially be an image built from the Dockerfile above?

If so, would it make sense to put an image on dockerhub so people can use it out of the box without building one? (I can put one up somewhere).

Reply:
Ya, just the image built from Dockerfile. Let me make the changes and test everything again.

If you put the image out publicly, everyone pulling it will have the same SSH key. I don't really see this as a problem since that is for inner pod connections within one's own cluster. Alternatively, we can generate the SSH key inside `head.yml`, but then one would need to paste the `pubkey` to `worker.yml`. What do you think?

Reply:
Can you elaborate on what the key is for? We didn't seem to need it in https://github.com/robertnishihara/ray-kubernetes.

Reply:
I use it so `ray-head` can `ssh` into workers to either install new dependencies or copy and paste necessary files without updating the `Dockerfile` and rebuilding the image
Same reason as the Manual Large Cluster Instructions on the main documentation found [here](https://ray.readthedocs.io/en/latest/using-ray-on-a-large-cluster.html)